tests/aem_hw.pm

```diff
@@ -489,19 +489,9 @@ sub setup_aem {
 }
 
 sub handle_luks_pass {
-    # Workaround for the bug where video signal is missing
-    if (check_var("LUKSPASS_NO_VIDEO_WORKAROUND", "1")) {
-        sleep 15;
-        for my $i (1..10) {
-            if (check_screen("luks-prompt", timeout => 10)){
-                last;
-            }
-            send_key 'esc';
-            send_key 'esc';
-            sleep 5;
-        }
-    }
-    assert_screen "luks-prompt", timeout => 30;
+    lukspass_no_video_workaround("luks-prompt");
+    # AEM boot on Intel platforms can take significant amount of time.
+    assert_screen "luks-prompt", timeout => 120;
```

Comment:
Commit message says 90s, please update the commit message.

Reply:
It was 90s at first, updated: https://github.com/3mdeb/openqa-tests-qubesos/pull/13/commits/eeac643e710d3392c68d73f9b0689a8e750d2d85